pkg/asset/installconfig/aws/session.go

```diff
@@ -198,12 +202,14 @@ func getUserCredentials() error {
 
 	creds, err := ini.Load(path)
 	if err != nil {
-		if os.IsNotExist(err) {
-			creds = ini.Empty()
-			creds.Section("").Comment = "https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html"
-		} else {
-			return err
+		if !os.IsNotExist(err) {
+			if err2 := os.Remove(path); err2 != nil {
+				logrus.Error(errors.Wrap(err2, fmt.Sprintf("failed to overwrite invalid file %s", path)))
+				return err
```

Comment:
Should this return `err`, `err2` or both?

Reply:
Was thinking of logging err2 and returning err.